docs/Customizing_site.rst

```diff
@@ -4153,10 +4156,12 @@ displayed on the top navigation bar.
 |image52|
```

Comment:
We should update this part to use the new shared layouts.

Reply:
removed